objectstorage/boltdb/boltdb.go

```diff
@@ -0,0 +1,220 @@
+package boltdb
+
+import (
+	"bytes"
+	"sync"
+
+	"github.com/iotaledger/hive.go/objectstorage"
+	"go.etcd.io/bbolt"
+)
+
+// Storage implements the ObjectStorage Storage interface around a BoltDB instance.
+type Storage struct {
+	instance *bbolt.DB
+	bucket   []byte
+}
+
+// New creates a new Storage with the underlying BoltDB.
+func New(db *bbolt.DB) objectstorage.Storage {
+	return &Storage{
+		instance: db,
+	}
+}
+
+func (s *Storage) WithRealm(realm []byte) objectstorage.Storage {
```

Comment:
Comment

Reply:
Why comment the implementation of an interface? the comment is in the interface